src/presentation/ui/page/ssls.templ

```diff
@@ -0,0 +1,314 @@
+package page
+
+import (
+	"github.com/speedianet/os/src/domain/entity"
+	componentForm "github.com/speedianet/os/src/presentation/ui/component/form"
+	componentStructural "github.com/speedianet/os/src/presentation/ui/component/structural"
+)
+
+script SslsIndexLocalState() {
+	document.addEventListener('alpine:init', () => {
+		Alpine.data('ssls', () => ({
+			// Primary states
+			sslPair: {},
+			resetPrimaryStates() {
+				this.sslPair = {
+					id: '',
+					virtualHostsHostnames: [],
+					certificate: '',
+					key: '',
+				}
+			},
+			init() {
+				this.resetPrimaryStates();
+			},
+
+			// Auxiliary states
+			get shouldDisableImportSslCertificateSubmitButton() {
+				return this.sslPair.virtualHostsHostnames.length == 0 || this.sslPair.certificate == '' || this.sslPair.key == ''
+			},
+			shouldImportSslCertificateAsFile: false,
+			get shouldDisableRemoveVirtualHostsHostnamesSubmitButton() {
+				return this.sslPair.id == '' || this.sslPair.virtualHostsHostnames.length == 0
+			},
+			downloadPemFile(isKeyFileContent) {
+				let pemFileContent = this.sslPair.certificate;
+				let fileExtension = 'crt';
+				if (isKeyFileContent) {
+					pemFileContent = this.sslPair.key
+					fileExtension = 'key';
+				}
+
```

Comment:
I would add a comment here to explain that JS does not allow for a direct blob download for reasons just Datena knows.

Reply:
Done!